docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+services:
+  rstudio:
+    build:
+      context: ./
+      dockerfile: Dockerfile-dev
+      args:
+        rstudioPort: 8787
+        rVersion: 3.6.2
+    container_name: iatlas-rstudio
+    ports:
+      - 8787:8787
```

Comment:
Please use the set_environment_variables script to create an environment variable for the port available to the user.

ie - 
```
ports:
      -${RSTUDIO_PORT}:8787
```